makeTrainSet.py

```diff
@@ -6,7 +6,7 @@
 
 import supervisedEtl as etl
 
-depth = 'deep'
+depth = 'udeepwide'
```

Comment:
What is `udeepwide`? It doesn't appear in the README for sgs (“<depth> is one of wide, deep, or udeep”).

I raised in an earlier review the issue of having this hard-coded in various source files. Is the expectation that the user will know which strings they have to edit to process the data they are interested in? That seems like a bad idea, but I think it's out of scope for @AstroVPK to fix it on this ticket. However, the sgs README should certainly be updated to describe whats happening.